core/src/desul/atomics/Compare_Exchange_SYCL.hpp

```diff
@@ -8,84 +8,79 @@ SPDX-License-Identifier: (BSD-3-Clause)
 
 #ifndef DESUL_ATOMICS_COMPARE_EXCHANGE_SYCL_HPP_
 #define DESUL_ATOMICS_COMPARE_EXCHANGE_SYCL_HPP_
-#include "desul/atomics/Common.hpp"
-#include "desul/atomics/SYCLConversions.hpp"
+
 #include <CL/sycl.hpp>
 
+#include "desul/atomics/Common.hpp"
+#include "desul/atomics/SYCLConversions.hpp"
```

Comment:
Is there a good reason to revert the order? I would rather avoid including `#include <CL/sycl.hpp>` into the other include files through this header. If they need it, they should include it themselves.

Reply:
I will disable clang-format for the header includes